examples/transfer-lamports/src/example.ts

```diff
@@ -0,0 +1,197 @@
+    (log.info('[step 1] Creating a transaction message'), createTransactionMessage({ version: 0 })),
+    /**
+     * Every transaction must state from which account the transaction fee should be debited from,
+     * and that account must sign the transaction. Here, we'll make the source account pay the fee.
+     */
+    tx => (
+        log.info({ address: SOURCE_ACCOUNT_SIGNER.address }, '[step 1] Setting the fee payer'),
+        setTransactionMessageFeePayer(SOURCE_ACCOUNT_SIGNER.address, tx)
+    ),
+    /**
+     * A transaction is valid for execution as long as it includes a valid lifetime constraint. Here
+     * we supply the hash of a recent block. The network will accept this transaction until it
+     * considers that hash to be 'expired' for the purpose of transaction execution.
+     */
+    tx => (
+        log.info(latestBlockhash, '[step 1] Setting the transaction lifetime'),
+        setTransactionMessageLifetimeUsingBlockhash(latestBlockhash, tx)
+    ),
+    /**
+     * Every transaction needs at least one instruction. This instruction describes the transfer.
+     */
+    tx =>
+        appendTransactionMessageInstruction(
+            /**
+             * The system program has the exclusive right to transfer Lamports from one account to
+             * another. Here we use an instruction creator from the `@solana-program/system` package
+             * to create a transfer instruction for the system program.
+             */
+            (log.info(
+                '[step 1] Creating an instruction to transfer Lamports from',
+                SOURCE_ACCOUNT_SIGNER.address,
+                'to',
+                DESTINATION_ACCOUNT_ADDRESS,
+            ),
+            getTransferSolInstruction({
+                amount: lamports(1_000_000n),
+                destination: DESTINATION_ACCOUNT_ADDRESS,
+                source: SOURCE_ACCOUNT_SIGNER,
+            })),
+            tx,
+        ),
+);
+
+/**
+ * STEP 2: SIGN THE TRANSACTION
+ * In order to prove that the owner of the account from which the tokens are being transferred
+ * approves of the transfer itself, the transaction will need to include a cryptographic signature
+ * that only the owner of that account could produce. We have already loaded the account owner's
+ * key pair above, so we can sign the transaction now.
+ */
+log.info('[step 2] Signing the transaction');
+const signedTransaction = await signTransactionMessageWithSigners(transactionMessage);
+log.info({ signature: getSignatureFromTransaction(signedTransaction) }, '[step 2] Transaction signed');
+
+/**
+ * STEP 3: SEND AND CONFIRM THE TRANSACTION
+ * Now that the transaction is signed, we send it to an RPC. The RPC will relay it to the Solana
+ * network for execution. The send method will resolve when the transaction is reported to have been
+ * confirmed, or will reject in the event of an error, or a timeout if the transaction lifetime is
+ * thought to have expired.
```

Comment:
At a minimum:

```suggestion
 * network for execution. The `sendAndConfirm` method will resolve when the transaction is reported to have been
 * confirmed, or will reject in the event of an error, or a timeout if the transaction lifetime is
 * thought to have expired.
```

...but I think maybe it's worth elaborating a bit on send vs. confirm and how this method actually works? Not super deep detail, but at least referencing how confirmation is tied to the lifetime setup from two steps ago.